src/test/java/com/codeborne/selenide/webdriver/EdgeDriverFactoryTest.java

```diff
@@ -35,10 +35,11 @@ void headless() {
     assertThat(options.get("extensions")).isEqualTo(emptyList());
 
     Map<String, Object> prefs = prefs(options);
-    assertThat(prefs).hasSize(7);
+    assertThat(prefs).hasSize(9);
     assertThat(prefs.get("credentials_enable_service")).isEqualTo(false);
     assertThat(prefs.get("profile.password_manager_enabled")).isEqualTo(false);
     assertThat(prefs.get("profile.password_manager_leak_detection")).isEqualTo(false);
+    assertThat(prefs.get("autofill.profile_enabled")).isEqualTo(false);
```

Comment:
Should we add `assertThat(prefs.get("autofill.credit_card_enabled")).isEqualTo(false);` too?

Reply:
Good point. Fixed.